src/routes/safe/components/Apps/components/AppFrame.tsx

```diff
@@ -51,12 +39,14 @@ const OwnerDisclaimer = styled.div`
 const AppWrapper = styled.div`
   display: flex;
   flex-direction: column;
-  height: 100%;
+  height: calc(100% + 59px);
```

Comment:
59px is the header, right?

Reply:
It's the footer, I decided to push it out of the viewport instead of hiding it